services/apps/alcs/src/alcs/search/application/application-advanced-search.service.ts

```diff
@@ -121,39 +118,64 @@ export class ApplicationAdvancedSearchService {
     const promises: Promise<{ fileNumber: string }[]>[] = [];
 
     if (searchDto.fileNumber) {
-      this.addFileNumberResults(searchDto, promises);
+      const promise = APP_SEARCH_FILTERS.addFileNumberResults(
+        searchDto,
+        this.applicationRepository,
+      );
+      promises.push(promise);
     }
 
     if (searchDto.legacyId) {
       this.addLegacyIDResults(searchDto, promises);
```

Comment:
Can you explain again why certain filters aren't being added to APP_SEARCH_FILTERS? Eg: legacyId, regionCode, decisionDate, etc

Reply:
I only moved filters that are shared, legacyId is only searchable in ALCS for example so I left it in the alcs search service.